packages/contracts-bedrock/snapshots/storageLayout/AnchorStateRegistry.json

```diff
@@ -27,25 +27,39 @@
     "slot": "1",
     "type": "contract IDisputeGameFactory"
   },
-  {
-    "bytes": "20",
-    "label": "portal",
-    "offset": 0,
-    "slot": "2",
-    "type": "contract IOptimismPortal2"
-  },
   {
     "bytes": "20",
     "label": "anchorGame",
     "offset": 0,
-    "slot": "3",
+    "slot": "2",
```

Comment:
Hang on. 
We said that its OK to have no spacers, because we're deploying a [whole new ASR](https://github.com/ethereum-optimism/optimism/blob/1040c51538d088c814b04352949f9f8297d743ef/packages/contracts-bedrock/src/L1/OPContractsManager.sol#L553).
But the new ASR is deployed in Upgrade 13. 
Are we deploying yet another new ASR for each chain in Upgrade 14 as well?

Reply:
We're invalidating all existing games anyway (to avoid porting over the blacklisting/invalidation state from the OptimismPortal) so there's really no benefit to keeping the existing `AnchorStateRegistry`. Deploying a new one avoids the spacer tech debt.

Reply:
OK, just to be clear then: yes we're deploying a new ASR in both 13 and 14. 

Fine with me if so.

Reply:
Yes, in both 13 and whatever upgrade this code ends up going into (14?)